examples/mlflow_artifacts/README.md

```diff
@@ -0,0 +1,34 @@
+This directory contains a set of files for demonstrating the mlflow artifacts service.
```

Comment:
This is an end-to-end example to make sure everything works correctly.

Reply:
This is *extremely* cool!